wallet/tickets.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/decred/dcrwallet/walletdb"
 )
 
+// TicketVoteBitsUpdate is a struct containing updates to ticket vote bits
+// settings in the stake database.
+type TicketVoteBitsUpdate struct {
```

Comment:
I can't do this, but then updates will be for a list of tickets and a list of voteBits/voteBitsExt.

I could also simply nest the voteBits/voteBitsExt struct inside the update struct. The voteBits/voteBitsExt struct should probably be located out in the stake package.